### DISCUSSION TITLE
ask for feature

### DISCUSSION THREAD

**Discussion by soheilamiri** [microsoft/terminal#12883] [Q&A] [answered]
hello dear team
Terminal Preview is nice and handy tool for me. 
i want to ask you to two feature to this tool 
1- add Broadcasting feature to allow inter command to multiply windows at a same time.
2- allow to resize the windows separator to right,left or up and down.
thanks

**Comment by zadjii-msft** [comment, marked as answer]
Thanks for the suggestion! Please make sure to search for duplicates when filing new issues. These are already being tracked by issues on our repo: 

1. see #2634
2. Are you talking about the pane separators? Those are already movable with actions: https://docs.microsoft.com/en-us/windows/terminal/panes#resizing-a-pane. If you want to be able to do that with a mouse, see #992.
